pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -40,18 +41,21 @@ var (
 )
 
 type CsiClusterConfigEntry struct {
```

Comment:
Seems like this struct should be deleted and the CSI struct referenced instead. But as you said, this can be a separate PR.

Reply:
Plus one